src/hexer/nifcgen.nim

```diff
@@ -1296,9 +1296,15 @@ proc importSymbol(e: var EContext; s: SymId) =
           transformInlineRoutines(e, c)
           return
 
-      e.dest.add tagToken("imp", c.info)
+      var dst = createTokenBuf(50)
```

Comment:
That's just a hack. The real solution here would be `if usesHeaderOrNodecl(pragmas): return`

Reply:
I rewrote it to use `prag.flags`.